src/DashboardService.Data/Interfaces/IPriorityRepository.cs

```diff
@@ -0,0 +1,17 @@
+using LT.DigitalOffice.DashboardService.Models.Db;
+using LT.DigitalOffice.Kernel.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.DashboardService.Data.Interfaces;
+
+[AutoInject]
+public interface IPriorityRepository
+{
+  Task<Guid?> CreateAsync(DbPriority dbDepartment);
+  Task<List<DbPriority>> GetAllAsync();
+  Task<DbPriority> GetAsync(Guid id);
+  Task<bool> EditAsync(Guid id);
```

Comment:
patch request was lost

Reply:
editAsync will be used for patch request 
![image](https://github.com/LT-Students/DashboardService/assets/42372955/bbacb1f5-e0d5-46cf-b427-c537726405f8)

Reply:
Based on what model will changes be made to the repository? Only the id is sent to the repository

Reply:
++